src/main/resources/db/migration/common/V1_218__node_register_missing_fk_and_seq.sql

```diff
@@ -0,0 +1,18 @@
+-- v47
+-- changes were not kept update to date for the node_register table so its being done now with the new
+-- changes for the table in v47.
+CREATE SEQUENCE node_seq START WITH 10000;
```

Comment:
Should this be called `der_seq`?

Reply:
No we are going forward with node_register table.